scripts/zones/Balgas_Dais/mobs/Phoedme.lua

```diff
@@ -0,0 +1,24 @@
+-----------------------------------
+-- Area: Balga's Dais
+--  Mob: Phoedme
+-- BCNM: Charming Trio
+-----------------------------------
+---@type TMobEntity
+local entity = {}
+
+entity.onMobInitialize = function(mob)
+    mob:setMod(xi.mobMod.CHARMABLE, 0)
```

Comment:
This is a mobMod

Use `mob:setMobMod(xi.mobMod.CHARMABLE, 0)`

Same in the others.